Assets/Post Processing/ApplyPostProcessing.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Photon.Pun.Demo.PunBasics;
+using UnityEngine.Rendering.PostProcessing;
+
+public class ApplyPostProcessing : MonoBehaviour
+{
+    PlayerManager playerManager;
+    public Vignette vignetteLayer;
+    Grayscale grayscaleLayer;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        playerManager = transform.parent.GetComponent<PlayerManager>();
+        PostProcessVolume volume = transform.GetComponent<PostProcessVolume>();
+        volume.profile.TryGetSettings(out vignetteLayer);
+        volume.profile.TryGetSettings(out grayscaleLayer);
```

Comment:
would be nice to have some comments here, cause it's really hard to understand what's going on